parsec/utils/argv.c

```diff
@@ -351,9 +349,11 @@ char *parsec_argv_join_range(char **argv, size_t start, size_t end, int delimite
     for (p = &argv[start], i=start; *p && i < end; ++p, ++i) {
         str_len += strlen(*p) + 1;
     }
+    if (0 == str_len) {
```

Comment:
Why returning a copy of an empty string instead of NULL ? It is not like this function is heavily used, but why do we need to change the logic ?

Reply:
you did that as part of the first commit addressing coverity errors. I suppose we free the value later on without checking for NULL?

Reply:
I don't see why I did that, it turns out that function is never used.